packages/astro/src/assets/types.ts

```diff
@@ -284,3 +285,6 @@ export type RemoteImageProps<T> =
 			 */
 			inferSize?: false | undefined;
 	  });
+
+type SvgProps = astroHTML.JSX.SVGAttributes;
+export type SvgComponent = ((props: SvgProps) => any) & ImageMetadata;
```

Comment:
```suggestion
export type SvgComponent = ((props: astroHTML.JSX.SVGAttributes;) => any);
```

Reply:
Note: I didn't use GitHub UI to accept your suggestions, but rather just replicated them in my local and made a commit to ensure there are no syntax errors and it has proper formatting. :)